vitest.config.js

```diff
@@ -22,9 +24,12 @@ import { vitestSetupFilePath, getClarinetVitestsArgv } from "@hirosystems/clarin
 
 export default defineConfig({
   test: {
-    exclude: [ ...configDefaults.exclude, "example/**" ],
+    exclude: [...configDefaults.exclude, "example/**"],
     environment: "clarinet", // use vitest-environment-clarinet
-    singleThread: true,
+    pool: "forks",
```

Comment:
Vitest v1 has some breaking changes in the config schema.
Also, I experienced a few issues in some project with the "threads" pooling. While this project in particular didn't seems to have this issue, we now recommend to use forks instead of threads ([vitest doc](https://vitest.dev/config/#forks))